packages/components/src/focusable-iframe/index.tsx

```diff
@@ -3,12 +3,19 @@
  */
 import { useMergeRefs, useFocusableIframe } from '@wordpress/compose';
 import deprecated from '@wordpress/deprecated';
+/**
+ * Internal dependencies
+ */
+import type { FocusableIframeProps } from './types';
 
 /**
  * @param {Object}                                 props
```

Comment:
Should this be some variant of `ComponentPropsWithoutRef< 'iframe' >` to match my change? Idk enough about JSDoc types for an opinion. There are a few other good ways to do this: https://mortenbarklund.com/blog/react-typescript-props-spread/

Reply:
We usually remove the types entirely from JSDocs when using TypeScript types. In this case, since the JSDoc comment was just there for adding type information, I'd argue that we can just simply delete it altogether.